### PR TITLE
Add the missing step of creating PVC

### PR DESCRIPTION
**Pull request by piyush-garg**
This will add the missing step of creating PVC
and did some other refactoring according
to the recent changes of pipeline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ In short, in order to create a pipeline, one does the following:
 * Create custom or install [existing](https://github.com/tektoncd/catalog) reusable `Tasks`
 * Create a `Pipeline` and `PipelineResources` to define your application's delivery pipeline
 * Create a `PipelineRun` to instantiate and invoke the pipeline
+* Create a `PersistentVolumeClaim` to provide the volume/filesystem for pipeline execution.
```
Comment (ppitonak) [inline README.md]: this should go somewhere before PipelineRun

File: README.md [hunk 2]
```diff
@@ -339,19 +331,15 @@ frontend and backend image resource to the correct url with your namespace name
...
-NAME        TYPE    DETAILS
-api-repo    git     url: http://github.com/openshift-pipelines/vote-api.git
-ui-repo     git     url: http://github.com/openshift-pipelines/vote-ui.git
-api-image   image   url: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-api:latest
-ui-image    image   url: image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-ui:latest
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml
 ```
 
-A `PipelineRun` is how you can start a pipeline and tie it to the git and image resources that should be used for this specific invocation. You can start the pipeline using `tkn`:
+A `PipelineRun` is how you can start a pipeline and tie it to the persistentVolumeClaim and params that should be used for this specific invocation. You can start the pipeline using `tkn`:
```
Comment (ppitonak) [inline README.md]: There are two sentences introducing the command, omit one of them

File: README.md [hunk 3]
```diff
@@ -366,6 +354,8 @@ In order to track the pipelinerun progress run:
 tkn pipelinerun logs build-and-deploy-run-z2rz8 -f -n pipelines-tutorial
 ```
 
+Similarly, start a pipeline to build and deploy frontend application:
+
 ```bash
 $ tkn pipeline start build-and-deploy \
     -w name=shared-workspace,claimName=source-pvc \
```
Comment (ppitonak) [inline README.md:359]: `-p deployment-name=vote-api` is wrong

File: README.md [hunk 4]
```diff
@@ -426,7 +416,7 @@ $ oc get route vote-ui --template='http://{{.spec.host}}'
 ```
 
 
-If you want to re-run the pipeline again, you can use the following short-hand command to rerun the last pipelinerun again that uses the same pipeline resources and service account used in the previous pipeline run:
+If you want to re-run the pipeline again, you can use the following short-hand command to rerun the last pipelinerun again that uses the same pipeline resources, workspaces, params and service account used in the previous pipeline run:
```
Comment (ppitonak) [inline README.md]: I suggest to omit `pipeline resources` as we don't use them (althought the statement is techically correct).

**Review by ppitonak — CHANGES_REQUESTED** [review]

**Review by khrm — COMMENTED** [review]
/lgtm

**Review by ppitonak — APPROVED** [review]

**Comment by chmouel** [comment]
/lgtm
/approve

Thanks 👍🏼🔥

**Pull request merged**